config/default.yml

```diff
@@ -4575,6 +4575,7 @@ Style/SpecialGlobalVars:
   VersionAdded: '0.13'
   VersionChanged: '0.36'
   SafeAutoCorrect: false
+  CorrecAddRequireEnglish: true
```

Comment:
I guess we can name this simply "RequireEnglish" or "RequireEnglishLibrary".

Reply:
`RequireEnglish` sounds good to me. Naming things is never easy...